packages/next/types/index.d.ts

```diff
@@ -11,6 +11,14 @@ import {
 /// <reference types="react" />
 /// <reference types="react-dom" />
 
+declare namespace NodeJS {
+  interface ProcessEnv {
+    readonly NODE_ENV: 'development' | 'production'
+    readonly browser: boolean
```

Comment:
I feel like `browser` should not be defined, same for `crossOrigin`.

Reply:
Whoops -- these totally are meant for `Process` not `ProcessEnv`.